README.md

```diff
@@ -279,6 +279,21 @@ let json = to_json B
 (* "bbb" *)
 ```
 
+#### `[@@deriving json_string]`: a shortcut for JSON string conversion
+
+For convenience, one can use `[@@deriving json_string]` to generate converters
+directly to and from JSON strings:
+
+```ocaml
+type t = A [@@deriving json, json_string]
+
+let "\"A\"" = to_json_string A
+let A = of_json_string "\"A\""
```

Comment:
This snippet is a bit strange because of the escaped strings and the `let A` which makes it not valid OCaml. Maybe this?

```suggestion
type t = A [@@deriving json, json_string]

let json_str = to_json_string A (* "A" *)
let v = of_json_string "\"A\"" (* A *)
```

Reply:
it's a valid OCaml, though in case of string the match is not exhaustive.